tmt/utils/rest.py

```diff
@@ -201,6 +201,13 @@ def visit_literal(self, node: docutils.nodes.literal) -> None:
 
     depart_literal = _noop_departure
 
+    def visit_emphasis(self, node: docutils.nodes.literal) -> None:
+        self.log_visit(str(node))
+
+        self._emit(click.style(node.astext(), fg='yellow'))
```

Comment:
Just checked the `--help` output and it seems the emphasized string is duplicated:

    Additionally, one can set ref
    dynamically. This is possible using a
    special file in tmt format stored in the
    defaultdefault branch of a tests
    repository. This special file should
    contain rules assigning attribute
    ref in an adjustadjust
    block, for example depending on a test run
    context.

See `defaultdefault` there, one yellow and one black. @happz, any hint here?

Reply:
That would be a bug in ReST rendering. I have a fix in https://github.com/teemtee/tmt/pull/3519.